terraform/environments/ops/variables.tf

```diff
@@ -3,6 +3,11 @@ variable "project_id" {
   description = ""
 }
 
+variable "app_project_id" {
```

Comment:
if we intend to use stage content-api for e2e testing, i think we can straight up say it here for clarity, eg: `variable "stage_project_id"`

Reply:
I feel like the `stage`-vs-`prod` decision should be made by end-users.

(Most folks would use `stage`, but I can imagine a team that does **not** own the API itself using a `prod` API version.)

Reply:
I think this is a case where we would want to provide an opinion. 

If this is for testing the **website** end to end, we want to test against the most accurate version and functional deployment of the content API (prod). If we can't guarantee that the staging content-api is the latest and most accurate version, then any failure or passing of website e2e test will be questionable.

Reply:
alternatively, we could the e2e test set up and tear down a temporary content-api run service using the latest image. Probably a good idea, but for another PR

Reply:
Thanks - I changed `app_project_id` to `prod_project_id` and added the following note in `terraform/environments/ops/variables.tf`:
```
# Test against the `prod` version of the Content API
# - We care primarily about testing the **website** component, not the API itself.
# - If you want to test the API as well, you can use the `stage` API version instead.
```